collective.cpp

```diff
@@ -761,7 +761,7 @@ vector<Vec2> Collective::getBedPositions(const vector<PCreature>& creatures, con
   optional<SquareType> bedType = getDormInfo()[spawnType].getBedType();
   vector<Vec2> bedPos;
   for (int i : All(creatures))
-    for (int j : Range(30)) {
+    for (int __attribute__((unused)) j : Range(30)) {
```

Comment:
Way too much boilerplate, how about a macro, like this:
for (int UNUSED : Range(30))